deepmd/entrypoints/convert.py

```diff
@@ -17,20 +18,26 @@ def convert(
     output_model: str,
     **kwargs,
 ):
-    if FROM == "auto":
-        convert_to_21(input_model, output_model)
-    elif FROM == "0.12":
-        convert_012_to_21(input_model, output_model)
-    elif FROM == "1.0":
-        convert_10_to_21(input_model, output_model)
-    elif FROM in ["1.1", "1.2"]:
-        # no difference between 1.1 and 1.2
-        convert_12_to_21(input_model, output_model)
-    elif FROM == "1.3":
-        convert_13_to_21(input_model, output_model)
-    elif FROM == "2.0":
-        convert_20_to_21(input_model, output_model)
-    elif FROM == "pbtxt":
-        convert_pbtxt_to_pb(input_model, output_model)
+    if output_model[:-6] == ".pbtxt":
+        if input_model[:-6] != ".pbtxt":
+            convert_pb_to_pbtxt(input_model, output_model)
+        else:
+            raise RuntimeError("input model is already pbtx")
```

Comment:
pbtxt?

Reply:
I find converting model files to pbtxt useful, and I think this feature should be added to CLI.

Reply:
You wrote pbtx